tests/flag_util_test.py

```diff
@@ -292,5 +293,32 @@ def testBadYAML(self):
       self.parser.parse('{a')
 
 
+class MockFlag():
+  def __init__(self, name, value, present):
+    self.name = name
+    self.value = value
+    self.present = present
+
+
+class TestGetProvidedCommandLineFlags(unittest.TestCase):
+
+  def setUp(self):
+    flag_dict = {
+        'flag1': MockFlag('flag1', '1', True),
+        'flag2': MockFlag('flag2', '2', True),
+        'flag3': MockFlag('flag3', '3', False)
+    }
+    patcher = mock.patch(flag_util.__name__ + '.FLAGS')
+    flags_mock = patcher.start()
+    flags_mock.FlagDict.return_value = flag_dict
+    self.addCleanup(patcher.stop)
+
+  def testGetProvidedCommandLineFlags(self):
+    self.assertDictEqual({
```

Comment:
Do you need to make this an ordered dictionary so that it always passes unit tests? See https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/commit/6bd92986fa35e9456ef40a872c749292f66ded23#diff-44ccde18eb382a76acff381351dc2ef2R156